test/driver.c

```diff
@@ -42,10 +42,8 @@ check_error(mrb_state *mrb)
 static int
 eval_test(mrb_state *mrb)
 {
-  const char *prog = "report()";
-
   /* evaluate the test */
-  mrb_load_string(mrb, prog);
+  mrb_funcall(mrb, mrb_top_self(mrb), "report", 0);
```

Comment:
MRuby compiler codes(codegen.c and parse.y) is moved to mruby-bin-mrbc so `mrb_load_string` won't be defined by default.